clients/cobol-lsp-vscode-extension/src/__tests__/services/BridgeForGitSupport.spec.ts

```diff
@@ -35,6 +35,9 @@ jest.mock("vscode", () => ({
     getConfiguration: jest.fn().mockReturnValue({
       get: jest.fn(),
     }),
+    getWorkspaceFolder: jest
+      .fn()
+      .mockReturnValue({ uri: { fsPath: "C:\\my\\workspace" } }),
```

Comment:
not needed

Reply:
Why do you think so?

Reply:
Because it was not used by `loadProcessorsConfigForDocument` in the commit the comment was made on.